test/unit/modules/peers.js

```diff
@@ -166,6 +167,52 @@ describe('peers', function () {
 		});
 	});
 
+	describe('list', function () {
+
+		beforeEach(function (done) {
+			removeAll(done);
+		});
+
+		it('should list empty peers list when no peers were inserted before', function (done) {
+			peers.list({}, function (err, __peers) {
+				expect(__peers).to.be.an('array').and.to.have.lengthOf(0);
+				done();
+			});
+		});
+
+		it('should list the inserted peer after insert', function (done) {
+			peers.update(randomPeer);
+			peers.list({}, function (err, __peers) {
+				expect(__peers).to.be.an('array').and.to.have.lengthOf(1);
+				done();
+			});
+		});
+
+		it('should list the inserted peer as Peer instance with normalized parameter set to false', function (done) {
```

Comment:
`should list the inserted peer as object with normalized parameter set to false` ?